hydra-node/test/Hydra/Model.hs

```diff
@@ -512,7 +540,9 @@ instance
           outputs <- lift $ serverOutputs node
           case find headIsOpen outputs of
             Just _ -> pure ()
-            Nothing -> error "The head is not open for node"
+            Nothing -> failure "The head is not open for node"
```

Comment:
```suggestion
            Nothing -> error "The head is not open for node"
```

I think `error` provides a better output (it's ironic, but the model test suite does not utilize the more detailed `HUnitFailure`)